keras_cv/layers/preprocessing/random_cut_mix.py

```diff
@@ -0,0 +1,126 @@
+import tensorflow as tf
+import tensorflow.keras as keras
+from tensorflow.keras import layers
+
+
+class RandomCutMix(layers.Layer):
```

Comment:
I recommend renaming to `CutMix` since that's the name of the technique (e.g. we don't use `RandomDropout`)

Reply:
Done - good point, it also just flows better in sentences to drop the random.